src/plugins/intel_cpu/src/nodes/mkldnn_conv_node.cpp

```diff
@@ -1000,52 +1038,120 @@ void MKLDNNConvolutionNode::prepareParams() {
         pAttrLocal = initPrimitiveAttr();
     }
 
-    std::shared_ptr<MKLDNNDescriptor> desc = createMkldnnConvDesc(inMemoryDesc->getDnnlDesc(),
-                                                                  weightMemoryDesc->getDnnlDesc(),
-                                                                  outMemoryDesc->getDnnlDesc(),
-                                                                  biasDesc);
+    updatePadding();
+    ConvKey key = {inMemoryDesc,
+                   weightMemoryDesc,
+                   biasDesc,
+                   outMemoryDesc,
+                   stride,
+                   dilation,
+                   paddingL,
+                   paddingR,
+                   *pAttrLocal,
+                   selected_pd->getImplementationType()};
+
+    auto engine = getEngine();
+    auto builder = [&engine](const ConvKey& key) -> executorPtr {
+        auto createMkldnnConvDesc = [](const mkldnn::memory::desc& srcDesc,
+                                       const mkldnn::memory::desc& wghDesc,
+                                       const mkldnn::memory::desc& dstDesc,
+                                       DnnlMemoryDescCPtr biasDescPtr,
+                                       const std::vector<size_t>& stride,
+                                       const std::vector<ptrdiff_t>& dilation,
+                                       const std::vector<ptrdiff_t>& paddingL,
+                                       const std::vector<ptrdiff_t>& paddingR,
+                                       mkldnn::algorithm alg) -> std::shared_ptr<MKLDNNDescriptor> {
+            mkldnn::memory::desc dnnlBiasDesc;
+            if (biasDescPtr) {
+                // WA to align IR bias representation (3 to 5 rank tensors) to oneDNN representation (1 rank tensor)
+                dnnlBiasDesc = biasDescPtr->getDnnlDesc().reshape({dstDesc.dims()[1]});
+            }
 
-    auto itpd = desc->createPrimitiveDescriptorIterator(getEngine(), *pAttrLocal);
+            return std::make_shared<MKLDNNDescriptor>(createDescriptorInternal(srcDesc,
+                                                                               wghDesc,
+                                                                               dnnlBiasDesc,
+                                                                               dstDesc,
+                                                                               (biasDescPtr != nullptr),
+                                                                               stride,
+                                                                               dilation,
+                                                                               paddingL,
+                                                                               paddingR,
+                                                                               alg));
+        };
```

Comment:
M.b. it's a matter of taste, but I think we can replace this lambda with the part of bias desc creation code:
```cpp
    mkldnn::memory::desc dnnlBiasDesc;
    if (biasDescPtr) {
        // WA to align IR bias representation (3 to 5 rank tensors) to oneDNN representation (1 rank tensor)
        dnnlBiasDesc = biasDescPtr->getDnnlDesc().reshape({dstDesc.dims()[1]});
    }
```
And use the `dnnlBiasDesc` variable in the two calls of `createDescriptorInternal` below the code frame. In the proposed case, the code will be more compact and readable.